packages/file-search/src/node/file-search-service-impl.spec.ts

```diff
@@ -24,36 +25,33 @@ testContainer.load(new ContainerModule(bind => {
     bind(FileSearchServiceImpl).toSelf().inSingletonScope();
 }));
 
-describe('search-service', function () {
-
-    this.timeout(10000);
-
-    it('shall fuzzy search this spec file', async () => {
+describe('search-service', () => {
+    test('shall fuzzy search this spec file', async () => {
         const service = testContainer.get(FileSearchServiceImpl);
         const rootUri = FileUri.create(path.resolve(__dirname, "..")).toString();
         const matches = await service.find('spc', { rootUri });
         const expectedFile = FileUri.create(__filename).displayName;
         const testFile = matches.find(e => e.endsWith(expectedFile));
-        expect(testFile).to.be.not.undefined;
-    });
+        expect(testFile).toBeDefined();
+    }, 10000);
 
-    it('shall respect nested .gitignore');
-    //     const service = testContainer.get(FileSearchServiceImpl);
-    //     const rootUri = FileUri.create(path.resolve(__dirname, "../../test-resources")).toString();
-    //     const matches = await service.find('foo', { rootUri, fuzzyMatch: false });
+    test('shall respect nested .gitignore', async () => {
```

Comment:
Please note, the previous construct printed `pending`. What does the current one do? Does it report a skipped or passed test?

Reply:
Ah, I will fix it later. `test.skip` is how we do in Jest.